docs/source/contributor-guidelines.rst

```diff
@@ -0,0 +1,141 @@
+==================================
+Lookit Guidelines for Contributors
+==================================
+
+Interested in helping write the code which powers the Lookit platform?  Thanks for supporting open source science!  This page describes the process any would-be contributor should plan to use.  We have included some beginner-friendly details in case you are new to open source projects.
+
+At a high level, you should plan to make feature-specific branches off of the `develop` branch of a local copy of the code running on your own machine.  This will keep the codebase as clean as possible.  Before submitting a PR, merge in the most recent changes from the `develop` branch.  
+
+System dependencies
+~~~~~~~~~~~~~~~~~~~
+
+Runs on Python 3.6 w/ dependencies listed in `requirements/defaults.txt`
+
+To run this project locally, fork and clone the repo, download the project dependencies, and set up a superuser on your local machine via these steps:
+
+Fork and clone the repo
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Fork the project on GitHub and git clone your fork, e.g.:
+
+SSH::
+
+    git clone <username>@github.com:<username>/lookit-api.git
+    
+HTTPS::
+
+    git clone https://github.com/<username>/lookit-api.git
+
+Start a `virtualenv`
+~~~~~~~~~~~~~~~~~~~~
+
+Replace `/path/to/python3` with the appropriate filepath::
+
+    $ cd lookit-api
+    $ virtualenv -p /path/to/python3 lenv
+    $ . lenv/bin/activate
+
+You may want to configure a global .gitignore on your machine and include your virtualenv along with any files specific to your system.  A sample global .gitignore is available here https://gist.github.com/octocat/9257657 -- you can tell git to globally ignore a .gitignore file via::
+
+    git config --global core.excludesfile ~/path/to/your/.gitignore_global
+
+Install dependencies
+~~~~~~~~~~~~~~~~~~~~
+
+The project dependencies listed in `requirements/defaults.txt` can be installed with pip::
+
+    (lenv) $ pip install --upgrade pip
+    (lenv) $ pip install -r requirements/defaults.txt
```

Comment:
Instead of including detailed directions for lookit-api here, reference instructions for running either django app (lookit-api) or also ember app (ember-lookit-frameplayer) locally; can move helpful details there but should probably not have installation directions in both places.